0011-interledger-payment-request/0011-interledger-payment-request.md

```diff
@@ -2,21 +2,23 @@
 
 ## Preface
 
-This document specifies the Interledger Payment Request (IPR) data format. It is intended for use in protocols in which the recipient of an Interledger payment first communicates a request for payment to the sender.
-
-This document provides an example flow to illustrate how IPR could be used in a higher level protocol. This document also details a method for generating Interledger transfer conditions that enable recipients to use the condition as an integrity check on incoming transfers and packets, without maintaining state about all outstanding payment requests.
+This document specifies the Interledger Payment Request (IPR) transport protocol. It is an end-to-end protocol in which the recipient of an Interledger payment first communicates a request for payment to the sender. This document also details how the recipient can verify that incoming payments match previously created requests without storing a history of all outstanding requests.
```

Comment:
> ## Preface
> This document specifies the Interledger Payment Request (IPR) transport protocol. It is an end-to-end protocol...

I suggest you drop the "Preface" header and change the opening sentences to read:

> The Interledger Payment Request (IPR) transport protocol is an end-to-end protocol...